lessons/lesson1.md

```diff
@@ -1,5 +1,28 @@
-## Lesson 1: Bootstrap
+## Lesson 1: Create a Console Integration 
 
-Bootstrap your app.
+In order to use custom event SDK, you need to get below informtion from console integration.
 
-Next lesson: [Creating your app](lesson2.md)
\ No newline at end of file
+- `IMS Org Id`: The Organization Id for which the provider, event metadata, etc are to be created which can be obtained using the Console or Transporter API.
+- `API key`: The API Key ( client id ) for the integration ( project workspace ) 
+- `JWT Token`: Note that I/O Management Service needs to be enabled for the integration
+- `Config.zip`: Config file downloaded from console including private key and certificate_pub.crt
+- `project.json`: for instance, `projectname-orgId-Production.json` file downloaded from console 
+
+
+1. Navigate to Adobe I/O console at [https://console.adobe.io](https://console.adobe.io) in your browser and create a project or using your exsiting project 
+2. Select `Add to Project` -> `Add an API` -> `Adobe Services` -> `I/O managemenet API`
+![add-api](assets/add-api.png)
+
+3. follow the steps to configure API, create a new service account (JWT) credential, `config.zip` will be
```

Comment:
you may need to briefly explain the 2 options for generating new key pair or use existing one.